.github/workflows/release.yml

```diff
@@ -17,34 +17,69 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |-
           echo 'GITHUB_TOKEN=${{secrets.GITHUB_TOKEN}}' > .release-env
-      - name: draft release
+      - name: build and upload goreleaser binaries
         run: make release
+
+  rocksdb-x86_64:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    needs: ["goreleaser"]
```

Comment:
I've tried to run goreleaser and nix building in parallel, but goreleaser has an issue to update existing draft release: https://github.com/goreleaser/goreleaser/issues/3072